src/api/java/com/minecolonies/api/configuration/ServerConfiguration.java

```diff
@@ -171,8 +171,8 @@ protected ServerConfiguration(final ForgeConfigSpec.Builder builder)
         maxColonySize = defineInteger(builder, "maxColonySize", 20, 1, 250);
         minColonyDistance = defineInteger(builder, "minColonyDistance", 8, 1, 200);
         initialColonySize = defineInteger(builder, "initialColonySize", 4, 1, 15);
-        maxDistanceFromWorldSpawn = defineInteger(builder, "maxdistancefromworldspawn", 30000, 1000, Integer.MAX_VALUE);
```

Comment:
this max value I actually chose intentionally as most ppl do not adjust it and it causes better interactivity/closeness on servers. I'd recommend keeping it and instead making in the error message clear that this is a configurable value